pkg/notifications/teams.go

```diff
@@ -0,0 +1,50 @@
+package notifications
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+)
+
+func init() {
+	initers = append(initers, func(cfg map[string]string) Notifier {
+		if url, ok := cfg["teams_url"]; ok {
```

Comment:
Typical Go style is to return nil early:

    		if url, ok := cfg["teams_url"]; !ok {
    		    return nil
    		}
    		notifier := &teamsNotifier{

...

Reply:
Fixed and branch squashed